tortoise/state.go

```diff
@@ -434,7 +434,7 @@ type headerWithSign struct {
 	sign   sign
 }
 
-func decodeVotes(evicted, blid types.LayerID, base *ballotInfo, exceptions types.Votes) (votes, types.LayerID, error) {
+func decodeVotes(blid types.LayerID, base *ballotInfo, exceptions types.Votes) (votes, types.LayerID, error) {
```

Comment:
That seems suspicious, are we sure that `evicted` is not needed and just wasn't forgotten when implementing `decodeVotes`?